sentry-android/src/main/java/com/joshdholtz/sentry/Sentry.java

```diff
@@ -94,6 +98,7 @@
 	private String packageName;
 	private int verifySsl;
 	private SentryEventCaptureListener captureListener;
+	private Map<String, Object> contexts;
```

Comment:
Any idea on a better name for this? Context already means something else in Android programming, but the key in the Sentry event is `contexts` - https://docs.getsentry.com/hosted/clientdev/interfaces/#context-interfaces